core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html

```diff
@@ -53,6 +53,7 @@
                     !isOnTerminalCard()">
           <md-button aria-label="" class="oppia-learner-continue-button protractor-test-continue-to-next-card-button" focus-on="<[::CONTINUE_BUTTON_FOCUS_LABEL]>" ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)" translate="I18N_PLAYER_CONTINUE_BUTTON">
           </md-button>
+          <img class="logo-bottom-nav-bar" ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)" src="/assets/images/icons/626x626_continue_icon.png">
```

Comment:
Btw, note that you can generally right-click on an element and select Inspect Element (or similar, depending on browser) to view the HTML source code for the element in the DOM tree. This should allow you to correlate what's going on between the page you're seeing and the code that is producing it.